tests/unit/test_unit.py

```diff
@@ -142,14 +142,15 @@ def test_create_knowledge_bases(self, mock_post, mock_get):
 
         args, kwargs = mock_post.call_args
 
-        assert kwargs['headers'] == {'Authorization': 'Bearer ' + API_KEY}
+        assert kwargs['headers'] == {'Authorization': 'Bearer ' + API_KEY, 'Content-Type': 'application/json'}
 
         expected_create_request = {
             'name': test_knowledge_base_config.name,
             'description': test_knowledge_base_config.description,
             'vector_store': {
                 'engine': test_vector_store_config.engine,
-                'connection_data': test_vector_store_config.connection_data
+                'connection_data': test_vector_store_config.connection_data,
+                'table': test_vector_store_config.table
```

Comment:
ok, this was added here (but test skipped)  in https://github.com/mindsdb/minds_python_sdk/pull/63/files#diff-032aaf721a24a943fd1e19218ac0481328467c94e2debfa4e50d4182af0b1984R121